src/schema.js

```diff
@@ -6,16 +6,21 @@ import {
   GraphQLInt,
   GraphQLList,
 } from 'graphql';
+import { mutationWithClientMutationId } from 'graphql-relay';
 
 import Todo from './mongoose/todo';
 
 const TodoType = new GraphQLObjectType({
-  name: 'Todo',
+  name: 'TodoType',
   fields: () => ({
     _id: {
       type: GraphQLString,
       resolve: todo => todo.id,
     },
+    itemId: {
```

Comment:
you can just use `id` and `_id`.
`_id` is like a private id (generally, database id)
`id` is the global id, used by GraphQL/Relay to identify the nodes
https://facebook.github.io/relay/docs/en/graphql-object-identification.html

Reply:
see this: https://github.com/entria/graphql-dataloader-boilerplate/blob/master/src/type/UserType.js#L19